recipes/viennarna/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 
 build:
   skip: True # [osx]
```

Comment:
```suggestion
  skip: True # [osx and x86_64]
```
Right now it's skipping all osx. If you want osx-arm64 but not osx-64, you can specify it here. Otherwise, you can remove this line to build both.

Reply:
ouch!  Thanks for pointing it out!  I'll try